ci-operator/config/openshift/release/openshift-release-master__ci-4.16-upgrade-from-stable-4.15-from-stable-4.14.yaml

```diff
@@ -40,7 +40,7 @@ tests:
           cpu: 100m
           memory: 200Mi
     - ref: openshift-e2e-test
-    workflow: openshift-upgrade-aws-sdn
+    workflow: openshift-upgrade-aws-ovn
```

Comment:
This file is for 4.16, was this an intentional switch?

Reply:
:+1: does not make much sense to test something that's in no way supported